BasicSample/Assets/Shared/Scripts/MixedRealityProjectSelectionWindow.cs

```diff
@@ -0,0 +1,133 @@
+            m_selection = GUILayout.SelectionGrid(m_selection, MixedRealityProjectOptions, 1, EditorStyles.radioButton);
+            m_selectedMRConfiguration = (MixedRealityProjectConfiguration)m_selection;
+            if(GUILayout.Button("Apply"))
+            {
+                ApplySelectedConfiguration(m_selectedMRConfiguration);
+            }
+        }
+
+        private void ApplySelectedConfiguration(MixedRealityProjectConfiguration selectedMRConfiguration)
+        {
+            bool remoting = false;
+            BuildTargetGroup targetGroup;
+            switch(selectedMRConfiguration)
+            {
+                case MixedRealityProjectConfiguration.RunNativelyonHL2:
+                    targetGroup = BuildTargetGroup.WSA;
+                    EditorUserBuildSettings.SwitchActiveBuildTargetAsync(BuildTargetGroup.WSA, BuildTarget.WSAPlayer);
+                    EditorUserBuildSettings.wsaBuildAndRunDeployTarget = WSABuildAndRunDeployTarget.DevicePortal;
+                    EditorUserBuildSettings.wsaArchitecture = "ARM64";
+                    break;
+                case MixedRealityProjectConfiguration.RunNativelyonPCVR:
+                    targetGroup = BuildTargetGroup.Standalone;
+                    EditorUserBuildSettings.SwitchActiveBuildTargetAsync(BuildTargetGroup.Standalone, BuildTarget.StandaloneWindows64);
+                    break;
+                case MixedRealityProjectConfiguration.RunRemotelyonUWP:
+                    remoting = true;
+                    targetGroup = BuildTargetGroup.WSA;
+                    EditorUserBuildSettings.SwitchActiveBuildTargetAsync(BuildTargetGroup.WSA, BuildTarget.WSAPlayer);
+                    EditorUserBuildSettings.wsaBuildAndRunDeployTarget = WSABuildAndRunDeployTarget.LocalMachine;
+                    EditorUserBuildSettings.wsaArchitecture = "Intel64";
+                    // Player Capabilities
+                    UnityEditor.PlayerSettings.WSA.SetCapability(UnityEditor.PlayerSettings.WSACapability.InternetClient, true);
+                    UnityEditor.PlayerSettings.WSA.SetCapability(UnityEditor.PlayerSettings.WSACapability.InternetClientServer, true);
+                    UnityEditor.PlayerSettings.WSA.SetCapability(UnityEditor.PlayerSettings.WSACapability.PrivateNetworkClientServer, true);
+                    break;
+                case MixedRealityProjectConfiguration.RunRemotelyonWin32:
+                    remoting = true;
+                    targetGroup = BuildTargetGroup.Standalone;
+                    EditorUserBuildSettings.SwitchActiveBuildTargetAsync(BuildTargetGroup.Standalone, BuildTarget.StandaloneWindows64);
+                    break;
+                default:
+                    Debug.Log($"Could not find {selectedMRConfiguration}, setting default configuration as Standalone");
+                    targetGroup = BuildTargetGroup.Standalone;
+                    break;
```

Comment:
In this `default` case, do we need to continue changing settings? I'd expect we could just early return here, unless there's a reason we'd want to keep setting up the project?

Also, LogError or LogWarning could convey the severity better here

Reply:
looks like Debug.log is used across basic sample in other scene scripts as well. Not sure how else to log this here

Reply:
Fair enough; I'd use Debug.LogError just because I don't expect the app to reasonably be in this state.

More importantly, I still think we could just return here? There's no need to try and set anything up in this state, I think.

Reply:
I was looking at an old version of the file, nevermind!